extensions/rocksdb-repos/DatabaseContentRepository.cpp

```diff
@@ -80,6 +80,8 @@ bool DatabaseContentRepository::initialize(const std::shared_ptr<minifi::Configu
     logger_->log_error("NiFi Content DB Repository database open {} fail", directory_);
     is_valid_ = false;
   }
+
+  use_synchronous_writes_ = configuration->get(Configure::nifi_content_repository_rocksdb_use_synchronous_writes).value_or("true") == "true";
```

Comment:
I'd fall back to true on invalid values.
```suggestion
  use_synchronous_writes_ = configuration->get(Configure::nifi_content_repository_rocksdb_use_synchronous_writes).value_or("true") != "false";
```

Reply:
Good idea, updated in 157416366b6cd54caddea2d904690f4d4fa3f627